FARSI/src/ch03-02-data-types.md

```diff
@@ -1,48 +1,35 @@
-## Data Types
+## انواع داده ها
 
-Every value in Rust is of a certain *data type*, which tells Rust what kind of
-data is being specified so it knows how to work with that data. We’ll look at
-two data type subsets: scalar and compound.
+هر مقداری در راست از یک *نوع داده* خاص است، که به راست می گوید چه نوع داده ای مشخص شده است تا بفهمد که با آن داده چطور رفتار کند. ما به برسی دو نوع از زیر مجموعه های انواع داده ها خواهیم پرداخت: scalar (اسکالر) و compound (مرکب).
```

Comment:
i think you shouldn't translate the RUST because if you do, the meaning will reference something else like right or straight that way, instead of راست just write RUST.

Reply:
ok. thanks i will consider that